tests/scripts/testcases_run.sh

```diff
@@ -52,19 +52,19 @@ fi
 
 # Tests Cases
 ## Test Master API
-ansible-playbook --limit "all:!fake_hosts" tests/testcases/010_check-apiserver.yml $ANSIBLE_LOG_LEVEL
+ansible-playbook --limit "all:!fake_hosts" -e @${CI_TEST_VARS} tests/testcases/010_check-apiserver.yml $ANSIBLE_LOG_LEVEL
```

Comment:
why you change ci test scripts here ?

Reply:
@Miouge1 please check it

Reply:
Yup I notice this was missing from tests, some tests use variable from test file but those variable are never loaded.. Maybe I can do a separate pr.

Reply:
Good catch, this is definitely supposed to be there.